docs/release-notes.md

```diff
@@ -37,4 +39,7 @@ The toolkit enables robust scaling through:
 #### UI-Toolkit
 - **KVM freeze intermittently:** We have added a small delay in handling mouse interactions that prevents us from flooding the AMT channel.  There are still a few occasions where KVM could still freeze.  We are still root causing this new issue
 #### RPS
-- **Failure to add Device:** If an RPS call to add new device information and credentials to Vault fails, the RPS should revert the changes to the other service and emit an error message. Currently RPS will provide an error, but it does not revert the changes. Resolving the failing issue with the service and then re-running the activation restores the data integrity between the database and Vault.
\ No newline at end of file
+- **[Data shouldn't be added if vault calls fail](https://github.com/open-amt-cloud-toolkit/rps/issues/254):** Currently, if vault calls fail (i.e. during creation of domain), the domain is successfully added to the database despite not being added to vault. Ideally, the database entry should be rolled back if vault call fails.
```

Comment:
Currently, if vault calls fail (e.g., during the creation of domain), the domain is successfully added to the database despite not being added to vault. Ideally, the database entry should be rolled back if vault call fails.

Reply:
Done